pyteal/compiler/scratchslots.py

```diff
@@ -35,7 +35,7 @@ def collectScratchSlots(
 def assignScratchSlotsToSubroutines(
     subroutineMapping: Dict[Optional[SubroutineDefinition], List[TealComponent]],
     subroutineBlocks: Dict[Optional[SubroutineDefinition], TealBlock],
-) -> Dict[Optional[SubroutineDefinition], Set[int]]:
+) -> Tuple[Dict[Optional[SubroutineDefinition], Set[int]], Set[int]]:
     """Assign scratch slot values for an entire program.
```

Comment:
@algoidurovic Minor:  Can you update the docstring to reflect the return type change?